Content.Server/Extinguisher/FireExtinguisherSystem.cs

```diff
@@ -30,13 +32,24 @@ public override void Initialize()
         SubscribeLocalEvent<FireExtinguisherComponent, AfterInteractEvent>(OnAfterInteract);
         SubscribeLocalEvent<FireExtinguisherComponent, GetVerbsEvent<InteractionVerb>>(OnGetInteractionVerbs);
         SubscribeLocalEvent<FireExtinguisherComponent, SprayAttemptEvent>(OnSprayAttempt);
+        /// <summary>
```

Comment:
put xmldoc on the method itself not subscribing
also this has no reason to be documented the code is very simple